compute_tools/src/spec.rs

```diff
@@ -244,3 +244,21 @@ pub fn handle_databases(spec: &ClusterSpec, client: &mut Client) -> Result<()> {
 
     Ok(())
 }
+
+pub fn create_writablity_check_data(client: &mut Client) -> Result<()> {
```

Comment:
This doesn't look as something related to the spec json processing, so I'd put it into the same checker file you created